issuance/issuance.go

```diff
@@ -43,9 +41,11 @@ type ProfileConfig struct {
 	AllowSCTList    bool
 	AllowCommonName bool
 
-	Policies            []PolicyConfig `validate:"min=1,dive"`
 	MaxValidityPeriod   config.Duration
 	MaxValidityBackdate config.Duration
+
+	// Deprecated: we do not respect this field.
+	Policies []PolicyConfig `validate:"-""`
```

Comment:
Drive by review to appease the linter.
```suggestion
	Policies []PolicyConfig `validate:"-"`
```